### PR TITLE
Using poetry for package (dependency) management

### PR DESCRIPTION
**Pull request by elenbaasc**

### CODE REVIEW

**Comment by elenbaasc** [comment]
Can you check how/if building the binary and publishing to PyPI works through `poetry`?

File: pyproject.toml [hunk 1]
```diff
@@ -1,49 +1,56 @@
...
-]
-include-package-data = false
+[tool.poetry]
+name = "opensquirrel"
+version = "0.0.5"
+description = "A quantum circuit transformation and manipulation tool"
+authors = [
+    "Pablo Le Henaff <p.lehenaff@tudelft.nl>",
+    "Roberto Turrado Camblor <r.turradocamblor@tudelft.nl>",
+    "Chris Elenbaas <chris.elenbaas@tno.nl"
```
Comment (rturrado) [inline pyproject.toml:8]: Missing a `>` at the end of Chris' email.

Comment (elenbaasc) [reply pyproject.toml:8]: Oh wow, well noticed!

**Comment by elenbaasc** [comment]
I put those comments in there already as a reminder of the tools (and tool configurations) I want to add.

**Pull request merged**